install/common.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+function clone_and_archive_default_theme()
```

Comment:
Why do we want to keep a copy of the default theme ?

Reply:
So the default theme here is mutable ? I thought the theme being archived here was the inmutable version of the default theme.